test/verify/check-networking-vlan

```diff
@@ -41,9 +41,9 @@ class TestNetworking(NetworkCase):
         # Make a VLAN interface
         b.click("button:contains('Add VLAN')")
         b.wait_popup("network-vlan-settings-dialog")
-        b.click("#network-vlan-settings-dialog tr:contains('Parent') li[value='%s'] a" % iface, True)
-        b.set_val("#network-vlan-settings-dialog tr:contains('Name') input", "tvlan")
-        b.set_val("#network-vlan-settings-dialog tr:contains('VLAN Id') input", "123")
+        b.click("#network-vlan-settings-body form div ul li[value='%s'] a" % iface, True)
+        b.set_val("#network-vlan-settings-interface-name-input", "tvlan")
+        b.set_val("#network-vlan-settings-vlan-id-input", "123")
```

Comment:
This needs to be conditionalized on `m.image == "rhel-7-6-distropkg"` as that runs the old code (see [failing test](https://fedorapeople.org/groups/cockpit/logs/pull-10822-20190104-042525-0a28d740-verify-rhel-7-6-distropkg/log.html#256)). In that case it should run the old test code, too. Actually it's fairly amazing that most of the test changes do work on 7.6, too. Maybe the selector can also be changed to apply in both cases.